src/api/options-lifecycle.md

```diff
@@ -1,174 +1,174 @@
 # Options: Lifecycle {#options-lifecycle}
 
-:::info See also
-For shared usage of lifecycle hooks, see [Guide - Lifecycle Hooks](/guide/essentials/lifecycle)
+:::info Xem thêm
+Để sử dụng chung các lifecycle hooks, xem [Guide - Lifecycle Hooks](/guide/essentials/lifecycle)
 :::
 
 ## beforeCreate {#beforecreate}
 
-Called when the instance is initialized.
+Được gọi khi thực thể được khởi tạo.
 
-- **Type**
+- **Loại**
 
   ```ts
   interface ComponentOptions {
     beforeCreate?(this: ComponentPublicInstance): void
   }
   ```
 
-- **Details**
+- **Chi tiết**
 
-  Called immediately when the instance is initialized, after props resolution, before processing other options such as `data()` or `computed`.
+  Được gọi ngay khi thực thể được khởi tạo, sau khi xử lý props, trước khi xử lý các options khác như `data()` hoặc `computed`.
 
-  Note that the `setup()` hook of Composition API is called before any Options API hooks, even `beforeCreate()`.
+  Lưu ý rằng `setup()` của Composition API được gọi trước bất kỳ hook Options API nào, thậm chí là `beforeCreate()`.
 
 ## created {#created}
 
-Called after the instance has finished processing all state-related options.
+Được gọi sau khi thực thể đã hoàn thành việc xử lý tất cả các options liên quan đến state.
 
-- **Type**
+- **Loại**
 
   ```ts
   interface ComponentOptions {
     created?(this: ComponentPublicInstance): void
   }
   ```
 
-- **Details**
+- **Chi tiết**
 
-  When this hook is called, the following have been set up: reactive data, computed properties, methods, and watchers. However, the mounting phase has not been started, and the `$el` property will not be available yet.
+  Khi hook này được gọi, các thành phần sau đã được thiết lập: reactive data, computed properties, methods, và watchers. Tuy nhiên, giai đoạn mounted chưa được bắt đầu, và thuộc tính `$el` sẽ chưa có sẵn.
 
 ## beforeMount {#beforemount}
 
-Called right before the component is to be mounted.
+Được gọi ngay trước khi component được mounted.
 
-- **Type**
+- **Loại**
 
   ```ts
   interface ComponentOptions {
     beforeMount?(this: ComponentPublicInstance): void
   }
   ```
 
-- **Details**
+- **Chi tiết**
 
-  When this hook is called, the component has finished setting up its reactive state, but no DOM nodes have been created yet. It is about to execute its DOM render effect for the first time.
+  Khi hook này được gọi, component đã hoàn thành việc thiết lập reactive state, nhưng chưa có DOM nodes nào được tạo ra. Nó sẽ thực thi DOM render effect lần đầu tiên.
```

Comment:
được tạo ra. Nó đang chuẩn bị thực thi DOM render effect lần đầu tiên.
Chỗ này không biết 2 bạn như nào nhưng mình thấy hơi lấn cấn chữ nó. Nó là chỉ hook hay component.
About to -> đang chuẩn bị